cmd/helm/registry_logout.go

```diff
@@ -31,10 +31,11 @@ Remove credentials stored for a remote registry.
 
 func newRegistryLogoutCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	return &cobra.Command{
-		Use:   "logout [host]",
-		Short: "logout from a registry",
-		Long:  registryLogoutDesc,
-		Args:  require.MinimumNArgs(1),
+		Use:               "logout [host]",
+		Short:             "logout from a registry",
+		Long:              registryLogoutDesc,
+		Args:              require.MinimumNArgs(1),
+		ValidArgsFunction: noCompletions,
```

Comment:
Is it safe to obtain a list of registries for which the user is logged in by looking at the `$HELM_REGISTRY_CONFIG` file?  Or could registries be stored elsewhere (e.g., `~/.docker/config.json`).

It would be nice to complete the registry addresses that the user can actually logout from, if we can find an exhaustive list.

Reply:
To do this, I think we should use an ORAS API to obtain the list.  I haven't had time to look if such an API exists yet.

Reply:
Its not so straightforward, as certain registries may skip `login` altogether and use credential helpers

Reply:
Ok, so let's not provide any completions right now